addons/rook/1.10.6/install.sh

```diff
@@ -499,7 +499,9 @@ function rook_scale_down_ekco() {
     kubectl -n kurl scale deployment ekc-operator --replicas=0
     rook_did_scale_down_ekco=1 # local to caller
     log "Waiting for ekco pods to be removed"
-    spinner_until 120 ekco_pods_gone
+    if ! spinner_until 120 ekco_pods_gone; then
+       bail "Unable to scale down ekco operator"
```

Comment:
the issue now is that bail will exit 1 which is not the same as return 1 which will allow the caller to choose how to handle the return value

```suggestion
       logFail "Unable to scale down ekco operator"
       return 1
```